src/app/utils/Utils.ts

```diff
@@ -860,4 +861,19 @@ export class Utils {
     document.execCommand('copy');
     document.body.removeChild(element);
   }
+
+  public static isDevLandscape(): boolean {
+    const configService: ConfigService = new ConfigService();
```

Comment:
Should not be instantiated like this.
Inject the config service where needed and test the Landscape here or create an Utils Service that provides these helper methods which will in the end be used the same way to hide the tab.